web_src/less/_font_i18n.less

```diff
@@ -89,7 +89,7 @@ each(@fonts, {
     font-family: @family;
     src: @src;
     font-weight: @weight;
-    unicode-range: ~"U+11??, U+2E80-4DBF, U+4E00-9FFF, U+A960-A97F, U+AC00-D7FF, U+F900-FAFF, U+FE00-FE6F, U+FF00-FFEF, U+1F2??, U+2????";
+    unicode-range: U+11??, U+2E80-4DBF, U+4E00-9FFF, U+A960-A97F, U+AC00-D7FF, U+F900-FAFF, U+FE00-FE6F, U+FF00-FFEF, U+1F2??, U+2????;
```

Comment:
Not directly an issue but stylelint was unable to parse this `~` less syntax. I converted to plain CSS and confirmed that it still transpiles correctly.